htmd/adaptive/adaptiverun.py

```diff
@@ -276,29 +276,41 @@ def __setattr__(self, key, value):
         uisetattr(self, key, value, all)
 
     def _algorithm(self):
+
         logger.info('Postprocessing new data')
-        datalist = simlist(glob(path.join(self.datapath, '*', '')), glob(path.join(self.inputpath, '*', 'structure.pdb')),
-                           glob(path.join(self.inputpath, '*', '')))
+
+        if self.app.get_class() == 'PmemdLocal':
+            datalist = simlist(glob(path.join(self.datapath, '*', '')),
+                               glob(path.join(self.inputpath, '*', 'structure.prmtop')),
+                               glob(path.join(self.inputpath, '*', '')))
+        else:
+            datalist = simlist(glob(path.join(self.datapath, '*', '')),
+                               glob(path.join(self.inputpath, '*', 'structure.pdb')),
+                               glob(path.join(self.inputpath, '*', '')))
```

Comment:
I would prefer to make this more general without `if`s and class names. I think I will modify simlist to search automatically for structure files in the input folders if only input folders are passed without structure file.
```
            datalist = simlist(glob(path.join(self.datapath, '*', '')),
                                      glob(path.join(self.inputpath, '*', '')))
```
will automatically search in each input folder for a file with extension ['prmtop', 'psf', 'pdb'].